crates/core/main.rs

```diff
@@ -215,14 +210,10 @@ fn eprint_nothing_searched() {
 /// prints each path sequentially using a single thread.
 fn files(args: &Args) -> Result<bool> {
     let quit_after_match = args.quit_after_match()?;
-    let subject_builder = args.subject_builder();
     let mut matched = false;
     let mut path_printer = args.path_printer(args.stdout())?;
-    for result in args.walker()? {
-        let subject = match subject_builder.build_from_result(result) {
-            Some(subject) => subject,
-            None => continue,
-        };
+    let subjects = sorted_results(args);
+    for subject in subjects {
```

Comment:
Same deal here as for `search` above.

Reply:
also resolved by https://github.com/BurntSushi/ripgrep/pull/2361/commits/d4706913adf88cb3c6b395d0afcf493f39208169